ember-server/src/main/scala/org/http4s/ember/server/EmberServerBuilder.scala

```diff
@@ -156,6 +156,7 @@ final class EmberServerBuilder[F[_]: Concurrent: Timer: ContextShift] private (
       )
       _ <- Resource.make(Applicative[F].unit)(_ => shutdown.await)
       _ <- Resource.liftF(ready.get.rethrow)
+      _ <- Resource.liftF(logger.info(s"Ember-Server service bound to address: $bindAddress"))
```

Comment:
Blaze printed out a pretty crazy banner too, do we want to do something like that in Ember also?

Reply:
I vote yes.  I did it as a joke, and people absolutely loved it.  Like, some of the most positive feedback we've ever gotten.

THE PEOPLE DEMAND THEIR FIGLETS.

Reply:
(Unlike the potential deadlock I fretted about above, I don't want to block this on a figlet.)